packages/expect/src/__tests__/__snapshots__/to_throw_matchers.test.js.snap

```diff
@@ -28,6 +28,16 @@ Instead, it threw:
 <red>      <dim>at jestExpect (</>packages/expect/src/__tests__/toThrowMatchers-test.js<dim>:24:74)</></>"
 `;
 
+exports[`.toThrow() fails when given an empty string 1`] = `
+"<dim>expect(</><red>function</><dim>).toThrow(</><green>string</><dim>)</>
+
+Expected the function to throw an error matching:
+  <green>\\"\\"</>
+Instead, it threw:
+<red>  Error</>
```

Comment:
shouldn't this text include `'apple'`?

Reply:
The test file defines a custom error with a fixed message for the snapshot - I’ve just reused that which is why it doesn’t say apple

Reply:
seems silly, doesn't it? Is the error inconsistent if we drop the fixed message?

Reply:
Yeah it does a bit! I’ll give it a go and see what happens, maybe it was giving inconsistent line numbers :shrug:

Reply:
is it still valid?

Reply:
Also, `toThrowErrorMatchingSnapshot` only snapshots the `message`, not `stack`